framework/sysdeps/linux/kvm.c

```diff
@@ -574,7 +574,11 @@ int kvm_generic_run(struct test *test, int cpu)
                 default:
                     stop = 1;
                     result = EXIT_FAILURE;
-                    log_error("Unexpected exit reason: %d.\n", ctx.runs->exit_reason);
+                    if (ctx.runs->exit_reason == KVM_EXIT_FAIL_ENTRY)
```

Comment:
How about the other `KVM_EXIT_*` constants?

Reply:
Oh i thought i was supposed to only take care of the `KVM_EXIT_FAIL_ENTRY`. Let me work on all of them. I believe 29 of them that present in the `kvm_run` structure.